### PR TITLE
ib_bw_perf - v1

### PR DESCRIPTION
**Pull request by manvanthar**
fixes the below issues:
1. server data from /tmp/ib_log is now printed.
2. Disabled firewall before the test start, which is an prerequisite.

Signed-off-by: Manvanthara Puttashankar <manvanth@linux.vnet.ibm.com>

[test_run.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/766009/test_run.txt)
[job.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/766010/job.txt)

### CODE REVIEW

**Comment by narasimhan-v** [comment]
Looks good to me

File: io/net/infiniband/ib_bw_perf.py [hunk 1]
```diff
@@ -18,14 +18,19 @@
 # ib_write_bw test
 # ib_read_bw test
 # ib_atomic_bw test
+#
+#
+# Based on code by Manvanthara Puttashankar <manvanth@linux.vnet.ibm.com>
+# Copyright: 2015 IBM
+# https://github.ibm.com/ltctest/eio_src/blob/master/bin/infiniband/ib_fvt_test.sh
```
Comment (PraveenPenguin) [inline io/net/infiniband/ib_bw_perf.py]: I wouldn't call this a big change, would you? (the purpose of authorship is to mark significant changes ) example when the module is completely reworked or extended in a such extend that the original work is not that significant any more. 

See eg. the first response to http://programmers.stackexchange.com/questions/203014/author-has-inserted-copyright-into-code-with-gnu-public-license-notice-implica

Comment (PraveenPenguin) [inline io/net/infiniband/ib_bw_perf.py]: ok

File: io/net/infiniband/ib_bw_perf.py [hunk 2]
```diff
@@ -63,27 +68,42 @@ def setUp(self):
...
+            cmd = "systemctl stop firewalld"
+        elif detected_distro.name == "Suse":
+            cmd = "rcSuSEfirewall2 stop"
+        elif detected_distro.name == "centos":
+            cmd = "service iptables stop"
+        else:
+            self.skip("Distro not supported")
+        if process.system("%s && ssh %s %s" % (cmd, self.peer_ip, cmd),
+                          ignore_status=True, shell=True) != 0:
+            self.skip("Unable to disable firewall")
```
Comment (PraveenPenguin) [inline io/net/infiniband/ib_bw_perf.py]: ok

**Comment by PraveenPenguin** [comment]
@manvanthar thanks for patch looks good

**Comment by PraveenPenguin** [comment]
@narasimhan-v thanks for review

**Pull request merged**